src/sdk/deployer.ts

```diff
@@ -0,0 +1,71 @@
+import { ethers } from 'ethers';
+
+import { Strategy } from './strategy';
+
+export class Cohort {
+  public strategies: { [key: string]: Strategy };
+  private constructor(public provider: ethers.providers.Web3Provider) {
+    this.strategies = {};
+  }
+
+  public add(strategyLabel: string, strategy: Strategy) {
```

Comment:
Do we need to store the strategies in the deployer, or can the strategy object just be passed into the relevant function (`revoke`, `deploy`, `getEncrytpter` etc.) when needed? i.e. can we prevent the Deployer from having any state?

Reply:
Yeah I'm definitely +1 for removing state, the user can easily handle that themselves